data_converter/urls.py

```diff
@@ -76,6 +92,10 @@
     path('api/users/', include('users.urls')),
 
     path('api/', include(router.urls)),
+
+    url(r'^swagger(?P<format>\.json|\.yaml)$', schema_view.without_ui(cache_timeout=0), name='schema-json'),
+    url(r'^swagger/$', schema_view.with_ui('swagger', cache_timeout=0), name='schema-swagger-ui'),
+    url(r'^redoc/$', schema_view.with_ui('redoc', cache_timeout=0), name='schema-redoc'),
```

Comment:
use re_path instead url

Reply:
use 'schema/' prefix
```suggestion
    url(r'^schema/swagger(?P<format>\.json|\.yaml)$', schema_view.without_ui(cache_timeout=0), name='schema-json'),
    url(r'^schema/swagger/$', schema_view.with_ui('swagger', cache_timeout=0), name='schema-swagger-ui'),
    url(r'^schema/redoc/$', schema_view.with_ui('redoc', cache_timeout=0), name='schema-redoc'),
```